.github/workflows/nuget.yml

```diff
@@ -0,0 +1,252 @@
+name: NuGet Release
+
+on:
+    push:
+        branches:
+            - master
+            - 2.6.x
+    workflow_dispatch: {}
+
+concurrency:
+    group: nuget-${{github.ref}}
+    cancel-in-progress: true
+
+env:
+    DOTNET_CLI_TELEMETRY_OPTOUT: 1
+
+jobs:
+    windows:
+        name: Windows VS2022
+        runs-on: windows-2022
+
+        steps:
+            - name: Checkout CSFML
+              uses: actions/checkout@v4
+              with:
+                  path: CSFML
+
+            - name: Build x86
+              shell: pwsh
+              run: |
+                  cd CSFML/tools/nuget
+                  ./build.win.ps1 "win-x86"
+
+            - name: Build x64
+              shell: pwsh
+              run: |
+                  cd CSFML/tools/nuget
+                  ./build.win.ps1 "win-x64"
+
+            - name: Upload Artifact
+              uses: actions/upload-artifact@v4
+              with:
+                  name: CSFML-win-x86-x64
+                  path: CSFML/tools/nuget/CSFML/runtimes
+
+    linux-x64:
+        name: Linux x64
+        runs-on: ubuntu-22.04
+        container:
+            image: ubuntu:22.04
```

Comment:
Isn't this redundant?

Reply:
I guess so. This comes from the time, I tried to run everything through Docker.

Reply:
Hmm, I'll leave it, so I don't have to install the dependencies natively, but can just keep it in the shell script.